### ISSUE TITLE
no-resource-limits-pods rule uses limit instead of limits

### ISSUE THREAD

**Issue by mleneveut** [kubevious/kubevious#52]
### Describe the bug
The no-resource-limits-pods rule uses container.resources.limit instead of container.resources.limits

### To Reproduce
1. Go to 'Rule Editor'
2. Click on 'no-resource-limits-pods' rule
3. Go to 'Rule script' tab

### Environment Details:
 - Kubernetes Distribution/Version [e.g. GKE v1.13.12-gke.25, OpenShift 4.3] AKS v1.19.3
 - Browser [e.g. chrome, safari] Chrome
 - Version [e.g. v0.3] 0.8.15

**Comment by rubenhak** [comment]
@mleneveut, what kind of error are you seeing? Is it a red expandable error bar on the top? It would be great to see the error bar expanded as a screenshot, or just text copy. Also, please check for errors for **kubevious-backend** pod.

**Comment by rubenhak** [comment]
@mleneveut, fixed in version 0.9.9. Unfortunately since that is just a default, upgrade over existing deployments wouldn't fix the rule. Any new deployments would have the correct rule. Thanks for submitting the issue.

```
helm upgrade --atomic -i kubevious kubevious/kubevious --version 0.9.9 -n kubevious 
```
